lib/tasks/telemetry.rake

```diff
@@ -0,0 +1,48 @@
+require "foreman/telemetry"
+require "foreman/telemetry_sinks/noop_sink"
+require "foreman/telemetry_sinks/metric_exporter_sink"
+
+def initialize_telemetry
+  telemetry = Foreman::Telemetry.instance.setup(:metric_exporter, :prefix => SETTINGS[:telemetry][:prefix] || '')
+  load "config/initializers/5_telemetry_metrics.rb"
+  load Rails.root + "config/initializers/5_telemetry_ruby.rb"
+  telemetry
+end
+
+namespace :telemetry do
+  desc <<-END_DESC
+Telemetry tasks.
+END_DESC
+
+
+  task :metrics do
+    telemetry = initialize_telemetry
+    puts "| Metric name | Labels | Type | Description |"
```

Comment:
Tbh the output of this is very unreadable. What about just printing the info in yaml format instead?

Reply:
This is github/markdown compatible table output, the goal is to use this to generate and easily update documentation table.